.ci/Jenkinsfile

```
@@ -237,20 +244,30 @@ pipeline {
     success {
       whenTrue(!isPR() && params.notifyOnGreenBuilds) {
         slackSend(channel: "#${env.SLACK_CHANNEL}", color: 'good',
-                  message: "[${params.runTestsSuite}] Build Passed: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.RUN_DISPLAY_URL}|Open>).",
+                  message: "[${params.runTestsSuites}] Build Passed: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.RUN_DISPLAY_URL}|Open>).",
                   tokenCredentialId: 'jenkins-slack-integration-token')
       }
     }
     unsuccessful {
       whenFalse(isPR()) {
         slackSend(channel: "#${env.SLACK_CHANNEL}", color: 'danger',
-                  message: "[${params.runTestsSuite}] Build Failed: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.RUN_DISPLAY_URL}|Open>).",
+                  message: "[${params.runTestsSuites}] Build Failed: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.RUN_DISPLAY_URL}|Open>).",
                   tokenCredentialId: 'jenkins-slack-integration-token')
       }
     }
   }
 }
 
+def checkTestSuite(Map parallelTasks = [:], String suite, String tags) {
+  def regexps = [ "^e2e/_suites/${suite}/.*", "^.ci/.*", "^cli/.*", "^e2e/.*\\.go" ]
+  if ("${FORCE_SKIP_GIT_CHECKS}" == "true" || isGitRegionMatch(patterns: regexps, shouldMatchAll: false)) {
+    log(level: 'INFO', text: "Adding ${suite}:${tags} test suite to the build execution")
+    parallelTasks["${suite}_${tags}"] = generateFunctionalTestStep(suite: "${suite}", tags: "${tags}")
+  } else {
+    log(level: 'WARN', text: "The ${suite}:${tags} test suite won't be executed because there are no modified files")
+  }
+}
+
 def generateStep(Map params = [:]){
```

Since params is a globlal variable I'd say to change the variable name and the references

```suggestion
def generateStep(Map args = [:]){
```